setup.py

```diff
@@ -1,15 +1,19 @@
 from setuptools import setup, find_packages
+version = "1.0.0"
 setup(
   name="e3db",
-  version="1.0.0",
+  version=version,
   packages=find_packages(),
-  install_requires="""
-PyNaCl >= 1.1, < 2
-requests >= 2.18, < 3
-""",
+  install_requires=[
+    'PyNaCl >= 1.1, < 2',
+    'requests >= 2.18, < 3'
+  ],
   url = "https://github.com/tozny/e3db-python",
+  download_url = 'https://github.com/tozny/e3db-python/archive/{0}.tar.gz'.format(version),
```

Comment:
Is this something we have to manually build, or will tagging the repo build it automatically?

Reply:
Doing a git tag, then pushing the tag with create the archive on github automatically :)